feature/bookmarks/src/test/kotlin/com/google/samples/apps/nowinandroid/feature/bookmarks/BookmarksViewModelTest.kt

```diff
@@ -88,6 +90,54 @@ class BookmarksViewModelTest {
         val item = viewModel.feedUiState.value
         assertIs<Success>(item)
         assertEquals(item.feed.size, 0)
+        assertTrue(viewModel.shouldDisplayUndoBookmark)
+
+        collectJob.cancel()
+    }
+
+    @Test
+    fun whenResourceViewed_setResourcesViewed() = runTest {
+        val collectJob = launch(UnconfinedTestDispatcher()) { viewModel.feedUiState.collect() }
+
+        // Give
+        newsRepository.sendNewsResources(newsResourcesTestData)
+        userDataRepository.setNewsResourceBookmarked(newsResourcesTestData[0].id, true)
+        val itemBeforeViewed = viewModel.feedUiState.value
+        check(itemBeforeViewed is Success)
+        check(!itemBeforeViewed.feed.first().hasBeenViewed)
```

Comment:
Please use proper kotlin-test assertions instead of `check(Boolean)`

```suggestion
        assertIs<Success>(itemBeforeViewed)
        assertFalse(itemBeforeViewed.feed.first().hasBeenViewed)
```